inline_verifier.go

```diff
@@ -232,10 +235,23 @@ func (s *BinlogVerifyStore) Serialize() BinlogVerifySerializedStore {
 	return s.store.Copy()
 }
 
+type mismatchType string
+
+const (
+	MismatchColumnMissingOnSource mismatchType = "column missing on source"
+	MismatchColumnMissingOnTarget mismatchType = "column missing on target"
+	MismatchRowMissingOnSource    mismatchType = "row missing on source"
+	MismatchRowMissingOnTarget    mismatchType = "row missing on target"
+	MismatchContentDifference     mismatchType = "content difference"
+	MismatchChecksumDifference    mismatchType = "rows checksum difference"
```

Comment:
there's a difference, because we check both checksum of a whole row + compare each rows columns separately, so the `checksum difference` is for when the SQL query generating checksum for whole row doesn't match one in source, while the content difference is when we compare each single column of a row and encounter and issue there

Reply:
🤔 If i understand this right, then maybe we should rename these to `MismatchFieldDifference` and `MismatchRowChecksumDifference`?